### PR TITLE
Add Dependabot documentation

### PR DESCRIPTION
**Pull request by gigitux**
This PR adds documentation about Dependabot. Also, I added an `automations` folder in `docs/internal-developers` folder where we can add documentation for all the automations that we have in this repository.

Internal discussion -> `pdToLP-96-p2#comment-104`

### CODE REVIEW

File: docs/internal-developers/automatations/README.md
Comment (sunyatasattva) [inline docs/internal-developers/automatations/README.md]:
```suggestion
This directory contains documentation for all the automations configured in this repository.
```

File: docs/internal-developers/automatations/README.md
Comment (sunyatasattva) [inline docs/internal-developers/automatations/README.md]: I assume this was a copy-paste mistake? 🤔

File: docs/internal-developers/automatations/README.md
Comment (sunyatasattva) [inline docs/internal-developers/automatations/README.md]: Directory name is misspelled: `automatations` → `automations`

File: docs/internal-developers/automatations/README.md
Comment (sunyatasattva) [inline docs/internal-developers/automatations/README.md]: Does this need to be this long? Can't we make the columns way shorter?

File: docs/internal-developers/automatations/dependabot.md
Comment (sunyatasattva) [inline docs/internal-developers/automatations/dependabot.md]:
```suggestion
It is the responsibility of the porter to review these PRs weekly and merge/reject them.
```

Should we really make mention of the porter role here since this is the public repo and I don't see any other mention anywhere? If so, should we link to our team handbook? CC @Aljullu

**Review by sunyatasattva — CHANGES_REQUESTED** [review]

**Comment by gigitux** [comment]
Thanks for your suggestions 🙇 ! I updated the PR 👍

File: docs/internal-developers/automatations/dependabot.md
Comment (gigitux) [reply docs/internal-developers/automatations/dependabot.md]: Mmm, good point! Let's wait for @Aljullu feedback!

Comment (Aljullu) [reply docs/internal-developers/automatations/dependabot.md]: We do have some mentions to the porter in the release checklist. I think it's fine. I wouldn't link to the Team Handbook, though.

File: docs/internal-developers/automations/dependabot.md
Comment (sunyatasattva) [inline docs/internal-developers/automations/dependabot.md]:
```suggestion
This repository uses [`Dependabot`](https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates) to help with keeping all the package dependencies (NPM, Composer, GitHub Actions) up to date. Without this in place, it's very easy to let the package versions we're using go stale and end up with a backlog of chores for updating those in the future. It is essential to keep dependencies updated to avoid security problems and lower overall upgrade costs.
```

**Review by sunyatasattva — APPROVED** [review]
Last small typo and reword. I'm pre-approving so you can merge after fixing that. 🙏

**Pull request merged**
